types/inline-style-prefixer/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for inline-style-prefixer 3.0
+// Project: https://github.com/rofrischmann/inline-style-prefixer
+// Definitions by: Andrej Hazucha <https://github.com/ahz>
+//                 dpetrezselyova <https://github.com/dpetrezselyova>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default InlineStylePrefixer;
```

Comment:
That code is pretty confusing -- If writes to the `exports` object, but then it replaces that object with a different one. Kind of like this:
```js
var exports = {};
exports.default = 1;
exports = exports.default;
console.log(exports); // 1
```

So by the end, there's no `default` property.